compat/src/index.js

```diff
@@ -71,9 +71,12 @@ class ContextProvider {
  */
 function Portal(props) {
 	let wrap = h(ContextProvider, { context: this.context }, props.vnode);
-	render(wrap, props.container);
+	let container = props.container;
+	let temp = props.container.appendChild(document.createTextNode(''));
+	hydrate(wrap, container);
 	this.componentWillUnmount = () => {
-		render(null, props.container);
+		container.removeChild(temp);
+		render(null, container);
```

Comment:
and this won't wipe the dom when it gets unmounted?

Reply:
That's a good remark, it was like this before aswell. I'll double check but it hasn't done that in tests yet.